docs/scale-cluster.rst

```diff
@@ -79,7 +86,8 @@ Finally, confirm with *Save*.
 The values corresponding to each scale unit will vary depending on the plan
 selected for the cluster. This also affects the cluster configuration and
 pricing. A warning will appear to remind the user that changing scale settings
-will be reflected in the charges billed.
+will be reflected in the charges billed. You will be able to see the new
+prices of *Compute* and *Storage* in the bottom right.
```

Comment:
isn't this more 'right' than 'bottom right' at that point? Terribly pedantic though so feel free to ignore me :)